Dockerfile

```
@@ -86,6 +86,14 @@ COPY --link caddy caddy
 COPY --link internal internal
 COPY --link testdata testdata
 
+# install fswatch (necessary for file watching)
+ARG FSWATCH_VERSION='1.17.1'
+WORKDIR /usr/local/src/fswatch
+RUN curl -L https://github.com/emcrisostomo/fswatch/releases/download/$FSWATCH_VERSION/fswatch-$FSWATCH_VERSION.tar.gz > fswatch.tar.gz  && \
+    tar xzf fswatch.tar.gz
```

```suggestion
RUN curl -L https://github.com/emcrisostomo/fswatch/releases/download/$FSWATCH_VERSION/fswatch-$FSWATCH_VERSION.tar.gz | tar xz
```

To prevent needlessly storing the archive in the cache layers.
 
(also, we use tabs not spaces in Dockerfiles)